src/sqlfluff/utils/reflow/sequence.py

```diff
@@ -6,7 +6,8 @@
 
 from sqlfluff.core.config import FluffConfig
 from sqlfluff.core.parser import BaseSegment, RawSegment
-from sqlfluff.core.rules.base import LintFix, LintResult
+from sqlfluff.core.rules import LintFix
+from sqlfluff.core.rules.base import LintResult
```

Comment:
```suggestion
from sqlfluff.core.rules import LintFix, LintResult
```

Same comment as above...